services/github/assets/in_progress.svg

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Not that it matters, but I usually like to feed SVGs through https://jakearchibald.github.io/svgomg/ to reduce their size and get rid of unneeded tags. For example, I was able to have the following reduction on this file :)

![screen shot 2019-02-19 at 8 02 48 am](https://user-images.githubusercontent.com/3766511/53016927-cf1ca380-341c-11e9-8672-7838b873303b.png)

Reply:
oh thanks for the tip!